components/builder-api-client/src/lib.rs

```diff
@@ -175,7 +181,10 @@ impl Client {
             .map_err(Error::HyperError)?;
 
         if res.status != StatusCode::NoContent {
-            debug!("Failed to promote group, status: {:?}", res.status);
+            debug!(
+                "Failed to promote or demote group, status: {:?}",
```

Comment:
It might be nice to state with certainty whether it was a promotion or a demotion that failed.